src/app/inkeep/useInkeepSettings.ts

```diff
@@ -0,0 +1,74 @@
+import type {
```

Comment:
The `app` folder is used for routing: https://nextjs.org/docs/app
Please see how GTM and Amplitude are integrated in the `src/client/analytics` and try applying the same pattern. Thanks!

Reply:
Moved to the client package now.